docs/purge-docker-registry.md

```diff
@@ -0,0 +1,48 @@
+# How-to purge docker registry
+
+This document explains the steps necessary to purge a docker registry.
+
+## Dependencies
+
```

Comment:
Why dont' you just define the steps below as makefile recipes and put everything inside a  ``Dockerfile``. It would be roughly something like

```Dockerfile
FROM alpine:latest

ENV DOCKER_USERNAME
ENV DOCKER_PASSWORD
ENV REGISTRY_URL

COPY docker-registry-curl.bash /scripts
COPY Makefile /scripts/Makefile

RUN apk add --no-cache \
   curl \
   jq \
   bash

ENTRYPOINT make --directory Makefile
```
Then you wrap the whole thing as with ``shellcheck.bash``, say ``registry-pruner-cli.bash``

```bash

IMAGE_NAME=local/docker-registry-purger:latest

if not exists docker images $IMAGE_NAME
  docker build --tag $IMAGE_NAME .
fi

exec docker run --env-file=.venv --rm --interactive $IMAGE_NAME "$@"
``` 

and then

1. `` registry-pruner-cli.bash list``
1. `` registry-pruner-cli.bash list target=YourRepo``
...

Reply:
This could even be a template to follow for all tooling!

Reply:
Well I think a doc is a minima.
A script would be good enough, makefile +dockerfile a luxury. But it's a good point.